Additional annotations needed static pods (#15385)
DataDog/documentation: content/en/containers/cluster_agent/endpointschecks.md

This feature is supported on Kubernetes for Agent v6.12.0+ and Cluster Agent v1.3.0+. Starting with v1.4.0, the Cluster Agent converts every Endpoints Check of a non-pod-backed endpoint into a regular Cluster Check. Enable the [Cluster Check][4] feature alongside Endpoints Checks to take advantage of this functionality.

> NOTE: If the pods behind your service are static pods, you'll need another annotation ad.datadoghq.com/endpoints.resolve. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See [an example][5] of this annotation with the kube API server.

Reviewer: ```suggestion
**Note:** If the pods behind your service are static pods, you need to add the annotation `ad.datadoghq.com/endpoints.resolve`. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See [this example][5] of the annotation with the Kubernetes API server.
```

Author: LGTM

> NOTE: If the pods behind your service are static pods, you'll need another annotation ad.datadoghq.com/endpoints.resolve. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See [an example][5] of this annotation with the kube API server.

Reviewer: ```suggestion
**Note:** If the pods behind your service are static pods, you need to add the annotation `ad.datadoghq.com/endpoints.resolve`. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See [this example][5] of using the annotation with the Kubernetes API server.
```

> NOTE: If the pods behind your service are static pods, you'll need another annotation ad.datadoghq.com/endpoints.resolve. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See an example of this annotation with the kube API server here - https://docs.datadoghq.com/containers/kubernetes/control_plane/?tab=helm#api-server-2

Reviewer: ```suggestion
**Note:** If the pods behind your service are static pods, add the annotation `ad.datadoghq.com/endpoints.resolve`. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See [this example][] of using the annotation with the kube API server. https://docs.datadoghq.com/containers/kubernetes/control_plane/?tab=helm#api-server-2
```
